cmd/nodecmd/create.go

```diff
@@ -291,11 +304,12 @@ func createNodes(cmd *cobra.Command, args []string) error {
 		networkoptions.NonLocalSupportedNetworkOptions,
 		"",
 	)
+	setGlobalNetworkFlags(network)
```

Comment:
why do we continue using globalNetworkFlags after the network is decided? most probably
we need to avoid using flags from here onwards

Reply:
I don't think we need this functon, the idea is to generate network from flags, not the other way